addons/html_builder/static/src/website_builder/plugins/form/utils.js

```diff
@@ -418,6 +418,9 @@ export function getDependencyEl(fieldEl) {
  */
 export function findCircular(dependentFieldEl, targetFieldEl) {
     const formEl = targetFieldEl.closest("form");
+    if (!formEl) {
+        return false;
+    }
```

Comment:
@shsa-odoo Adding timeout in this step solves the issue: https://github.com/odoo-dev/odoo/blob/master-mysterious-egg/addons/website/static/tests/tours/website_form_editor.js#L886

@FrancoisGe Do you think we should fix ```findCircular``` method for this case? the issue is produced when tests run in headless chrome, if we are running tour manually then issue is not producing.

Reply:
@msh-odoo Lets wait for @FrancoisGe ' s reponse on this. Meanwhile i will update the code for above remark.